pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -76,6 +76,12 @@ func (ctrl *Controller) syncNodeConfigHandler(key string) error {
 	if err := ctrl.cleanUpDuplicatedMC(managedNodeConfigKeyPrefix); err != nil {
 		return err
 	}
+	// explicitly setting the cgroupMode to "v2" and also updating the config node's spec if found empty
+	// This helps in updating the cgroupMode on all the worker nodes if they still have cgroupsv1 (Ex: RHEL8 workers)
+	if nodeConfig.Spec.CgroupMode == emptyInput {
+		nodeConfig.Spec.CgroupMode = osev1.CgroupModeDefault
```

Comment:
sorry I'm still seeing CgroupModeDeafult and emptyInput here (I am seeing the correct values below)

Reply:
also, why are we calling this code twice ? can you leave a comment why each situation is distinct?

Reply:
Sorry, My bad. Updated the Bootstrap section and forgot to update the original syncNodeConfig code. Updated now.

> also, why are we calling this code twice ? can you leave a comment why each situation is distinct?

Actually, this is how MCO is working where one set of changes are part of the rendering the machine configs during the bootstrap mode and all the related Bootstrap functions are required to have exactly similar changes to that of the original sync() functions.
cc: @yuqi-zhang